public/src/components/contact_list/contact_list_form.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { Field } from 'redux-form';
+import { Button, ButtonToolbar } from 'react-bootstrap';
+import FieldGroup from '../../components/common/form/field_group';
+import UploadField from '../../components/common/form/upload_field';
+
+export default class ContactListForm extends Component {
+  componentDidMount() {
+    if (!this.props.onSubmit) {
+      throw new Error('onSubmit is not passed. Pass in a function.');
+    } else if (!this.props.onCancel) {
```

Comment:
Was this to resolve an error? 
This is basically prop validation, yes?
[for reference:](https://facebook.github.io/react/docs/typechecking-with-proptypes.html)

Reply:
removed it for now 👍